tests/common/devices.py

```diff
@@ -326,6 +337,13 @@ def _get_os_version(self):
         output = self.command("sonic-cfggen -y /etc/sonic/sonic_version.yml -v build_version")
         return output["stdout_lines"][0].strip()
 
+    def _get_kernel_version(self):
+        """
+        Gets the SONiC kernel version
+        :return:
+        """
+        return self.setup()['ansible_facts']['ansible_kernel'].split('-')[0]
```

Comment:
Can you consider using command like `uname -r` to get kernel version? The ansible setup command is a little bit too heavy.